boltdesignsystem/bolt
# Feature/dropdown close on select

## Asana

https://app.asana.com/0/1126340469288208/1138736873218898/f

## Summary

When you click an item in the pathways dropdown (a new active pathway), the dropdown menu closes.

## Details

Tested this in Chrome (ShadowDom and no shadowDom), Firefox and IE11. 

## How to test

- Go to http://localhost:3000/pattern-lab/patterns/06-experiments-micro-journeys-combos--92-micro-journey-in-themes/06-experiments-micro-journeys-combos--92-micro-journey-in-themes.html
- Open the pathways dropdown menu.
- Click on a new pathway. The dropdown menu should then close.
